includes/documents/abstract-wcpdf-order-document.php

```diff
@@ -297,6 +297,25 @@ public function delete( $order = null ) {
 		do_action( 'wpo_wcpdf_delete_document', $this, $order );
 	}
 
+	public function regenerate( $order = null ) {
+		$order = empty( $order ) ? $this->order : $order;
+		if ( empty( $order ) ) {
+			return; //Nothing to update
+		}
+
+		//Get most current settings
```

Comment:
would there be a way to unify this with the normal `$this->get_settings()` method? It's good practice to prevent code duplication whenever possible.

Reply:
Since this is more of an optimization and doesn't change anything functionality wise, I'll open separate issue and merge this one